clients/lsp-markdown.el

```diff
@@ -40,23 +40,62 @@
   :package-version '(lsp-mode . "7.1"))
 
 (defcustom lsp-markdown-server-command-args '("--parser=remark-parse" "--stdio")
-  "commandline-arguments for the markdown lsp server."
+  "Command-line arguments for the markdown lsp server."
   :type '(repeat 'string)
   :group 'lsp-markdown
   :package-version '(lsp-mode . "7.1"))
 
+(defcustom lsp-markdown-remark-plugins [["#remark-preset-lint-markdown-style-guide"]]
```

Comment:
this should be a vector of strings not a vector of vectors of strings?

Reply:
The plugins are documented as array of arrays:

			"plugins": [
				["#retext-redundant-acronyms"],
				["#retext-overuse"],
				["#retext-intensify"],
				["#retext-repeated-words"],
				["#retext-spell", "#dictionary-en-gb"]
			]

https://github.com/unifiedjs/unified-language-server/blob/main/CONFIGURATION.md